code/modules/religion/rites.dm

```diff
@@ -59,6 +59,7 @@
 
 /datum/religion_rites/synthconversion/perform_rite(mob/living/user, obj/structure/altar_of_gods/AOG)
```

Comment:
what if you accidentally added dmdocs to this proc.... 😳

Reply:
https://github.com/tgstation/tgstation/pull/49874/files#diff-9a4cfd0b2e87282898698953f44176c7L14-L15